src/components/Radio/stories.js

```diff
@@ -1,46 +1,178 @@
 import React from "react";
+import { Form, Field } from "react-final-form";
 import { storiesOf } from "@storybook/react";
+import PropTypes from "prop-types";
+
+import Button from "../Button/index";
+import GridCol from "../GridCol/index";
+import GridRow from "../GridRow/index";
+import Layout from "../Layout/index";
+import MultiChoice from "../MultiChoice/index";
 import Radio from ".";
 
+const onSubmit = values => {
+  console.log(JSON.stringify(values, 0, 2));
+};
+
+const required = value => (value ? undefined : "Required");
+
+const RadioGroup = ({ label, hint, name, options, inline, ...input }) => (
+  <Layout>
+    <GridRow>
+      <GridCol>
+        <MultiChoice {...input} label={label} hint={hint}>
+          {options.map(o => (
+            <div key={o.value}>
+              <Radio
+                type="radio"
+                {...input}
+                inline={inline}
+                value={o.value}
+                checked={o.value === input.value}
+              >
+                {o.title}
+              </Radio>
+            </div>
+          ))}
+        </MultiChoice>
+      </GridCol>
+    </GridRow>
+  </Layout>
+);
+
+RadioGroup.defaultProps = {
+  hint: undefined,
+  inline: false,
+  name: undefined,
+  options: {}
+};
+
+RadioGroup.propTypes = {
+  label: PropTypes.string.isRequired,
+  hint: PropTypes.string,
+  name: PropTypes.string,
+  inline: PropTypes.bool,
+  options: PropTypes.shape({
+    title: PropTypes.string,
+    value: PropTypes.string
+  })
+};
+
+const App = () => (
+  <div>
```

Comment:
```<div>
 +    <Form
 +      onSubmit={onSubmit}
 +      render={({ handleSubmit, reset, submitting, pristine, values }) => (
 +        <form onSubmit={handleSubmit}>
 +          <Layout>
 +            <GridRow>
 +              <GridCol>```

I would suggest creating a util component that does this bit so we can repeat it elsewhere, maybe stick it in /stories/components/FinalFormWraper.js.